include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -230,6 +238,10 @@ inline void UDPSender::sendToDaemon(const std::string& message) noexcept {
     }
 }
 
+inline bool UDPSender::initialized() const noexcept {
```

Comment:
its too bad the UDP server and client cant share these bits of code, but at the very least the work the same way even if the code is duplicated